packages/react/src/components/FileUploader/FileUploader-story.js

```diff
@@ -87,6 +87,7 @@ const props = {
       ),
       buttonLabel: text('The button label (buttonLabel)', 'Add file'),
       buttonKind: buttonKind || 'primary',
+      disabled: boolean('Disabled (disabled)', false),
```

Comment:
We'll need to mirror this change into the v11 storybook as well through 
 `packages/react/src/components/FileUploader/next/FileUploader.stories.js`. I'll push a quick commit for it

Reply:
Added!

Reply:
thanks! didn't know you guys had a v11 storybook @tay1orjones